config/webpack.config.js

```diff
@@ -37,6 +37,7 @@ const config = {
           loader: 'babel-loader',
           options: {
             presets: ['@babel/preset-env'],
+            plugins: ['@babel/plugin-transform-runtime'],
```

Comment:
I had to manually install this since I don't think it got added to `package.json`.

Reply:
weird, it seems like its there...hrmmm